.github/workflows/atlas_build-and-push-docker.yml

```
@@ -0,0 +1,196 @@
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      # Process tags input to support both simple tags and advanced configuration
+      - name: Prepare tag configuration
+        id: tag_config
+        run: |
+          # Use TAGS_CONFIG environment variable to store tags configuration
+          echo "TAGS_CONFIG<<EOF" >> $GITHUB_ENV
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)